services/QuillLMS/app/controllers/stripe_integration/subscription_checkout_sessions_controller.rb

```diff
@@ -55,7 +65,17 @@ def create
     end
 
     private def success_url
-      "#{subscriptions_url}?checkout_session_id={CHECKOUT_SESSION_ID}"
+      if schools.empty? || !customer.admin?
```

Comment:
It looks to me like `.admin?` checks that the user role is an admin (I think that's defunct) and not that they have schools that they administrate.

Reply:
I think is a gotcha, since the other role booleans (student?, teacher?, staff?) behave as expected.

Reply:
Oops, you're right. I was looking at the [authentication function](https://github.com/empirical-org/Empirical-Core/blob/feat/admin-reports/services/QuillLMS/app/lib/quill_authentication.rb#L149).